app/web/src/features/dashboard/DashboardBanners.tsx

```diff
@@ -51,20 +52,19 @@ export default function DashboardBanners() {
                 {UPLOAD_PHOTO}
               </Typography>
               <Typography variant="inherit" paragraph>
-                <StyledLink to={routeToEditProfile()}>
-                  {CLICK_HERE_TO_EDIT}
-                </StyledLink>
+                <Button color="default" role="link">
```

Comment:
sorry for cut in on, but as I know putting <a> tag inside <button> tag is not valid html (some sources: https://www.w3schools.com/tags/tag_button.asp
https://stackoverflow.com/questions/51755195/is-it-a-valid-markup-to-put-some-tags-inside-button-tag )
maybe its better just to do onClick
```suggestion
                <Button color="default" role="link" onClick=(() => { history.push(routeToEditProfile()) })>
```

Reply:
Good catch, thank you.